pyspecProcScripts/integral_w_error.py

```diff
@@ -10,6 +10,7 @@ def integral_w_errors(
     sig_path,
     error_path,
     cutoff = 0.1,
+    frq_slice = None,
```

Comment:
By default it uses the automated integrate_limits function but for developing purposes we have the option of controlling exactly where to slice or how much/little signal to include